pkg/polaris/graphql/infinity-k8s/infinity_k8s.go

```diff
@@ -66,6 +66,13 @@ type AddK8sProtectionSetConfig struct {
 	RSType              string   `json:"rsType"`
 }
 
+// UpdateK8sProtectionSetConfig defines the input parameters required to update
+// a ProtectionSet.
+type UpdateK8sProtectionSetConfig struct {
```

Comment:
name can't be updated?

Reply:
no, since its also a crd and crd can't change names.